python/pyspark/pandas/frame.py

```diff
@@ -1292,7 +1292,7 @@ def applymap(self, func: Callable[[Any], Any]) -> "DataFrame":
         0   1.000000   4.494400
         1  11.262736  20.857489
 
-        You can omit the type hint and let pandas-on-Spark infer its type.
+        You can omit type hint and let pandas-on-Spark infer its type.
```

Comment:
You can omit type hint 
or 
You can omit type hints

Reply:
"the" is necessary there. Or else change to "type hints".